<launch>
    <node pkg="roomba_control" type="roomba_control_node" name="roomba_control" output="screen">
        <param name="world_frame" value="tag20"/>
        <param name="robot_frame" value="tag21" />
        <param name="state_update_fq" value="30" />

        <param name="position_controller/kp" value="0.00" />
        <param name="position_controller/kd" value="0.00" />
        <param name="position_controller/ki" value="0.00" />
        <param name="position_controller/minVal" value="-500" />
        <param name="position_controller/maxVal" value="500" />

        <param name="orientation_controller/kp" value="-5.0" />
        <param name="orientation_controller/kd" value="2.00" />
        <param name="orientation_controller/ki" value="0.00" />
        <param name="orientation_controller/minVal" value="-500" />
        <param name="orientation_controller/maxVal" value="500" />

        <param name="goal_tolerance" value="0.05"/>
        <param name="robot_index" value="1" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find roomba_control)/launch/roomba.rviz" />
   
</launch>
